code/game/gamemodes/changeling/evolution_menu.dm

```diff
@@ -421,6 +424,8 @@ var/list/sting_paths
 		if(hud_used)
 			hud_used.lingstingdisplay.icon_state = null
 			hud_used.lingstingdisplay.invisibility = 101
+			hud_used.lingchemdisplay.invisibility = 101
```

Comment:
Required since handle_changeling used to set this to 101 if you were not a cling. Refactor of handle_changeling can be done as well but I'll keep it out of the PR